app/controllers/reviews_controller.rb

```diff
@@ -56,6 +59,28 @@ def load_review
       flash[:danger] = t("review.review.fail")
       redirect_to reviews_path
   end
+
+  def like
+    @review=Review.all.find(params[:id])
```

Comment:
vì sao cần all

Reply:
Em muốn tìm trong phạm vi tất cả các review. Chỗ này em không dùng all mà dùng Review.find(params[:id] thì codacy nó báo lỗi "Unscoped find (and related methods) are a form of Direct Object Reference."